faqs/gtn/loading_data.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How can i load data?
+area: introduction
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+
+## Video Tutorials
+
+* Quick help: [**Manupulating NGS data with Galaxy: Getting Data In**](/tutorials/ngs/#getting-ngs-data-in)
+* Full tutorial: [**Uploading data into Galaxy**](/tutorials/upload/)
+* Dataset Collections, including creation during Upload: [**Processing many samples at once with collections**](https://galaxyproject.org/tutorials/collections/)
+* FTP/FTPS tutorial: [**FTP Upload**](/ftp-upload/)
```

Comment:
where are these links pointing to?

Reply:
Thanks for this @beatrizserrano i this links and looks like the content had been transfered.

Reply:
but this is pointing to local pages in the Galaxy Community Hub that will not work when we move the FAQ to the GTN. So I'd suggest that we link to the tutorials in the GTN whenever available